data/json/obsoletion/obsolete_items.json

```diff
@@ -1163,12 +1163,5 @@
     "dispersion": 50,
     "recoil": 350,
     "effects": [ "COOKOFF", "NEVER_MISFIRES" ]
-  },
-  {
-    "type": "COMESTIBLE",
```

Comment:
Was this migrated in a different PR?

Reply:
> Was this migrated in a different PR?

Yeah, migration handled in the PR linked in the description